src/gui/README.md

```diff
@@ -644,6 +644,54 @@ To remove all the rulers:
 gui::clear_rulers
 ```
 
+### Add Label to Layout
+
+To add a label to the layout use the following command:
+
+Returns: name of the newly created label.
+
+```tcl
+gui::add_label
+    x y
+    text
+    [color]
+    [size]
+    [name]
```

Comment:
It would be nicer to use switches than optional positional arguments.

Reply:
I'll add a proper TCL cmd